src/components/drawerbar.css

```diff
@@ -0,0 +1,19 @@
+.picture {
+    position: absolute;
```

Comment:
position absolute should be used as a last resort. You can use display grid to arange everything you want. Here is a small guide on using it [https://css-tricks.com/snippets/css/complete-guide-grid/#aa-grid-template-areas](https://css-tricks.com/snippets/css/complete-guide-grid/#aa-grid-template-areas)